plugins/unicodelib/unicodelib.cpp

```diff
@@ -718,6 +719,48 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+UnicodeString excludeNthWord(RuleBasedBreakIterator& bi, UnicodeString const & source, unsigned n)
+{
+	UnicodeString processed;
```

Comment:
Efficiency: Think about the number of UnicodeString that are created - each one will clone the string on the heap.  I think 3 UnicodeStrings are created, and it is currently likely to clone the string 2 or 3 times, when 1 could be done instead.